### ISSUE TITLE
Support fully-remote Dask schedulers

### ISSUE THREAD

**Issue by JoshKarpel** [JoshKarpel/dask-chtc#24]
Right now, we support a workflow that looks like this:

![Screenshot from 2020-07-17 12-52-55](https://user-images.githubusercontent.com/7133863/87816141-75482500-c82c-11ea-8df1-1a672f6e32b7.png)

We'd like to be able to support this:

![Screenshot from 2020-07-17 12-53-37](https://user-images.githubusercontent.com/7133863/87816202-901a9980-c82c-11ea-9f7a-1b8614abd7aa.png)

This will require Dask-CHTC to be able to do spooled remote submits. There will be networking and security implications for this, including the need to acquire HTCondor IDTOKENs. Presumably we could have a CLI command to help out with that.

This is related to #5, because a goal of this workflow is to be able to gather workers from many different places and hook them all up to your central scheduler.

I think we also need to understand how this impacts communication overheads - its possible that some workflows that rely on short tasks will be seriously impeded by running like this.

It would be very nice if we could support non-Linux platforms as well, but it will be annoying. We might be able to swing Windows with special install instructions, but Mac is non-starter at the moment. Hopefully we'll get improved HTCondor Python bindings distribution mechanisms soon-ish.

**Comment by JoshKarpel** [comment]
We ended up running into a major roadblock with the above plan: my home ISP was blocking incoming TLS connections. We can't really control this, so we need to rethink the approach. Braindump below...

The new plan is to do this:
![Screenshot from 2020-09-01 08-35-30](https://user-images.githubusercontent.com/7133863/91857867-20bb0680-ec2e-11ea-91e8-96f3651c1450.png)

The client will still be on the user's computer, the scheduler will be in Kubernetes, and it will remote-submit to a CHTC schedd. This will work because all the communications between the client and scheduler is from the client (i.e., they are outgoing connections, and will not be blocked by ISPs). The connections from the scheduler to the workers will be inside the CHTC network and won't be blocked.

The scheduler will need a remote-submit-enabled IDTOKEN, presumably generated by us on the user's behalf. I think that leads to the big question, which is: how do users "request" that the remote scheduler "service" be started up for them?
